clientv3/op.go

```diff
@@ -145,10 +155,22 @@ func WithPrefix() OpOption {
 		op.end = []byte{0}
 	}
 }
+
+// WithRange specifies the range of Get or Delete requests.
+// For example, Get requests with 'WithRange(end)' returns
+// the keys in the range [key, end).
 func WithRange(endKey string) OpOption {
 	return func(op *Op) { op.end = []byte(endKey) }
 }
+
+// WithFromKey specifies the range of Get or Delete requests
+// that keys are greater than or equal to key in the argument.
 func WithFromKey() OpOption { return WithRange("\x00") }
+
+// WithSerializable serializes Get and Delete requests. Which are
```

Comment:
this does not apply to delete request. delete request will always go through raft.